src/styles/utilities/textUnderline/types.ts

```diff
@@ -0,0 +1,5 @@
+import {CSSProperties} from 'react';
+
+type TextUnderlineStyles = Record<'textUnderlinePositionUnder' | 'textDecorationSkipInkNone', Partial<Pick<CSSProperties, 'textUnderlinePosition' | 'textDecorationSkipInk'>>>;
```

Comment:
Not a blocker: if we want to be more accurate I would just use:
```ts
type TextUnderlineStyles = {
    textUnderlinePosition: CSSProperties['textUnderlinePosition'];
    textDecorationSkipInkNone: CSSProperties['textDecorationSkipInk'];
}
```

Reply:
@fabioh8010 wdyt?

Reply:
LGTM!

Reply:
PR updated!